docs/01-app/01-getting-started/08-mutating-data.mdx

```diff
@@ -0,0 +1,308 @@
+---
+title: How to mutate data
+nav_title: Mutating Data
+description: Learn how to mutate data in your Next.js application.
+related:
+  title: API Reference
+  description: Learn more about the features mentioned in this page by reading the API Reference.
+  links:
+    - app/api-reference/functions/revalidatePath
+    - app/api-reference/functions/revalidateTag
+    - app/api-reference/functions/redirect
+---
+
+You can mutate data in Next.js using React's [Server Functions](https://react.dev/reference/rsc/server-functions). This page will go through how you can [create](#creating-server-functions) and [invoke](#invoking-server-functions) Server Functions.
```

Comment:
Shouldn't we still call them server actions? Maybe I am missing how we think about this. Server actions are the child of server functions, but specifically here there are all server actions and what we will continue to use for mutations.